frontend/styles/components/calendar/DailyViewRow.module.scss

```
@@ -1,9 +1,26 @@
   position: absolute;
   border-radius: 6px;
   z-index: 10;
 }
 
+// Background container spanning an overlapping cluster's full time range, rendered beneath
+// the cluster's meeting cards and the "+N" pill (z-index 10/12 below) -- no left-accent
+// border, unlike a real meeting card, since it isn't standing in for any one meeting.
+.clusterContainer {
+  position: absolute;
+  top: 0;
+  // Height is set inline (DailyViewRow.tsx) as an explicit pixel value, not here --
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)